experimental/dds/tree2/src/test/feature-libraries/sequence-field/invert.spec.ts

```diff
@@ -215,23 +217,21 @@ describe("SequenceField - Invert", () => {
 		it("return-from + redundant return-to => skip + skip", () => {
 			const input = [
 				Mark.returnFrom(1, brand(0), { isDstConflicted: true, changes: childChange1 }),
-				Mark.returnTo(1, brand(0), {
-					inverseOf: tag1,
-					cellId: { revision: tag2, localId: brand(0) },
-				}),
```

Comment:
This test data was wrong before: it did not represent a muted mark.
The test expectations were wrong too and have been updated.